homeassistant/components/tesla_fleet/__init__.py

```diff
@@ -173,14 +179,16 @@ async def _get_access_token() -> str:
                 )
                 continue
 
-            api = tesla.energySites.create(site_id)
+            api_energy: EnergySite = tesla.energySites.create(site_id)
```

Comment:
Not sure why we need to retype this, I'd expect mypy to infer the type here

Reply:
Removed.